### PR TITLE
redux to redux-server-rendering

### PR DESCRIPTION
**Pull request by robwise**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ gem 'react_on_rails', path: '../react_on_rails'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.4'
+gem 'rails', '4.2.5'
```
Comment (justin808) [inline Gemfile:7]: @robwise Let's get rid of this diff noise the next time we regenerate. We'll update the the branch https://github.com/shakacode/react_on_rails-generator-results/tree/fresh-rails-install first.

File: bin/rails [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env ruby
 begin
   load File.expand_path('../spring', __FILE__)
-rescue LoadError
+rescue LoadError => e
```
Comment (justin808) [inline bin/rails:4]: Just diff noise. Ignore.

File: bin/rake [hunk 3]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env ruby
 begin
   load File.expand_path('../spring', __FILE__)
-rescue LoadError
+rescue LoadError => e
```
Comment (justin808) [inline bin/rake:4]: Just diff noise. Ignore.

File: app/views/hello_world/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
-<h1 class="alert alert-info this-works">Client Rendering</h1>
+<h1 class="alert alert-info this-works">Server Rendering</h1>
 <%= react_component("HelloWorldApp",
                      @hello_world_props,
                      generator_function: false,
-                     prerender: false,
+                     prerender: true,
```
Comment (justin808) [inline app/views/hello_world/index.html.erb:5]: This is the key option for the rails helper!

File: Procfile.dev [hunk 5]
```diff
@@ -1,4 +1,4 @@
 web: rails s
 hot: sh -c 'cd client && npm start'
 client: sh -c 'rm app/assets/javascripts/generated/* || true && cd client && npm run build:dev:client'
-
+server: sh -c 'cd client && npm run build:dev:server'
```
Comment (justin808) [inline Procfile.dev:4]: Note, we have to rebuild the server webpack file with `wepback -w`. See definition [here](https://github.com/shakacode/react_on_rails-generator-results/blob/redux-to-redux-server-rendering-comparison-1_1_1/client/package.json#L32):

``` javascript
"build:dev:server": "webpack -w --config webpack.server.rails.config.js",
```

File: Gemfile (hunk 1 above)
Comment (robwise) [inline Gemfile:7]: @justin808 that's the weird thing—this is a diff between redux and redux-server-rendering, so it's nothing to do with the differences from the fresh-rails-install, right?

**Pull request closed without merging**